catalog/dags/providers/provider_workflows.py

```diff
@@ -196,6 +197,10 @@ def __post_init__(self):
         start_date=datetime(2020, 1, 1),
         ingester_class=BrooklynMuseumDataIngester,
     ),
+    ProviderWorkflow(
+        start_date=datetime(2023, 12, 10),
```

Comment:
Since this is monthly by default, this means it won't start until the beginning of the new year. It might be worth having it start this month!
```suggestion
        start_date=datetime(2023, 11, 30),
```

Reply:
Fixed in 232d0de.